backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -126,9 +126,17 @@ jQuery(function($) {
         ransack_value = $this.val();
       }
 
-      label = label.text() + ': ' + ransack_value;
+      switch (label.text()) {
+        case 'Date Range':
+          label = 'Start: ' + ransack_value;
```

Comment:
We're having a bit of duplication here since `+ ransack_value` is repeated for each case statement.
Can we move it outside of the `switch` and use something like `label + ransack_value` in HTML?
Does it make sense?

Reply:
Good point. Fixing it.